datadog_checks_dev/tests/tooling/commands/test_dep.py

```diff
@@ -0,0 +1,130 @@
+                        {"python_version": "py3", "requires_python": None, "yanked": False},
+                    ],
+                    "2.0.0": [
+                        {"python_version": "py2", "requires_python": None, "yanked": False},
+                        {"python_version": "py3", "requires_python": None, "yanked": False},
+                    ],
+                },
+            },
+            {'protobuf': {'py2': '2.0.0', 'py3': '2.0.0'}},
+            id='two versions',
+        ),
+        pytest.param(
+            {
+                "info": {"name": "protobuf"},
+                "releases": {
+                    "1.0.0": [
+                        {"python_version": "py2", "requires_python": None, "yanked": False},
+                        {"python_version": "py3", "requires_python": None, "yanked": False},
+                    ]
+                },
+            },
+            {'protobuf': {'py2': '1.0.0', 'py3': '1.0.0'}},
+            id='one versions',
+        ),
+        pytest.param(
+            {
+                "info": {"name": "protobuf"},
+                "releases": {
+                    "1.0.0": [
+                        {"python_version": "py3", "requires_python": ">=3", "yanked": False},
+                    ],
+                    "2.0.0": [
+                        {"python_version": "py3", "requires_python": ">=3", "yanked": False},
+                    ],
+                },
+            },
+            {'protobuf': {'py2': None, 'py3': '2.0.0'}},
+            id='no py2',
+        ),
+        pytest.param(
+            {
+                "info": {"name": "protobuf"},
+                "releases": {
+                    "1.0.0": [
+                        {"python_version": "py2", "requires_python": "<=3", "yanked": False},
+                    ],
+                    "2.0.0": [
+                        {"python_version": "py2", "requires_python": "<=3", "yanked": False},
+                    ],
+                },
+            },
+            {'protobuf': {'py2': '2.0.0', 'py3': None}},
+            id='no py3',
+        ),
+        pytest.param(
+            {
+                "info": {"name": "protobuf"},
+                "releases": {
+                    "1.0.0": [
+                        {"python_version": "py2", "requires_python": None, "yanked": False},
+                        {"python_version": "py3", "requires_python": None, "yanked": False},
+                    ],
+                    "2.0.0": [
+                        {"python_version": "py2", "requires_python": None, "yanked": True},
+                        {"python_version": "py3", "requires_python": None, "yanked": False},
+                    ],
+                },
+            },
+            {'protobuf': {'py2': '1.0.0', 'py3': '2.0.0'}},
+            id='yanked one py2 version',
+        ),
+    ],
+)
+@patch("aiomultiprocess.Pool.map", mock_map)
+@patch("aiohttp.ClientSession._request")
+@requires_py3
+@pytest.mark.asyncio
```

Comment:
Just curious, why does this need to be async?

Reply:
Because the function that is tested is async https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/dep.py#L178

Without this marker, the test would not work as expected. More info https://pypi.org/project/pytest-asyncio/